test/libyul/yulOptimizerTests/unusedStoreEliminator/covering_calldatacopy.yul

```diff
@@ -0,0 +1,59 @@
+{
+    let start := calldataload(0x10)
```

Comment:
This probably only makes sense with ``start`` being a constant in the latest version? Or is this test just for future reference?